tests/test_regressions.py

```diff
@@ -250,8 +250,8 @@ def test_issue193_splitting_function():
                  RETURN x;
                 END;
                 SELECT * FROM a.b;"""
-    splitted = sqlparse.split(sql)
-    assert len(splitted) == 2
+    statements = sqlparse.split(sql)
```

Comment:
the past tense of split is split -- https://simple.wiktionary.org/wiki/split

Since the result of sqlparse.split() is statements, it makes more sense to use that.